test/e2e/play_kube_test.go

```diff
@@ -2384,6 +2384,34 @@ var _ = Describe("Podman kube play", func() {
 		Expect(label).To(ContainSubstring("unconfined_u:system_r:spc_t:s0"))
 	})
 
+	It("--no-hostname", func() {
+		err := writeYaml(checkInfraImagePodYaml, kubeYaml)
+		Expect(err).ToNot(HaveOccurred())
+
+		kube := podmanTest.Podman([]string{"kube", "play", "--no-hostname", kubeYaml})
+		kube.WaitWithDefaultTimeout()
```

Comment:
mitr added a `podmanTest.PodmanExitCleanly()` for this kind of thing which we should start using. I wont repeat the comment but it would apply to several others in this change too.